app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+  def self.first_10(keyword_string)
+    @food_items = FoodService.search_10(keyword_string).map do |data|
```

Comment:
How could we maybe make a single method that gives us all the things that we want?


Here is an example:
```
def self.foods_by_keyword(keyword)
    foods_results = FoodService.foods_by_keyword(keyword)
    x = foods_results[:foods].map do |food_result|
      Food.new(food_result)
    end
    total_hits, foods = [foods_results[:totalHits], x]
  end
```